ocs_ci/deployment/acm.py

```diff
@@ -112,7 +112,10 @@ def download_binary(self):
             tempf.write(resp.content)
 
             # Actual submariner binary download
-            cmd = f"bash {tempf.name}"
+            if config.ENV_DATA.get("submariner_upstream_version_tag"):
+                cmd = f"VERSION={config.ENV_DATA.get('submariner_upstream_version_tag')} bash {tempf.name}"
+            else:
+                cmd = f"bash {tempf.name}"
```

Comment:
I would go with something like this:
```suggestion
            cmd = f"bash {tempf.name}"
            submariner_upstream_version_tag = config.ENV_DATA.get("submariner_upstream_version_tag")
            if submariner_upstream_version_tag:
                cmd = f"VERSION={submariner_upstream_version_tag} {cmd}"
```

Reply:
@petr-balogh , changed the approach using environ vars